test/presubmit-tests.sh

```diff
@@ -35,6 +35,20 @@ export GO111MODULE=on
 export KNATIVE_VERSION=${KNATIVE_VERSION:-latest}
 source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/presubmit-tests.sh
 
-# We use the default build, unit and integration test runners.
+# Run cross platform build to ensure kn compiles for Linux, macOS and Windows
+function run_cross_platform_build() {
```

Comment:
Call this post_submit_tests(), as explained in step 4 about the presubmits script in  https://github.com/knative/test-infra/blob/master/scripts/README.md as you don't want to run this for every build, unit and integration test run.

Reply:
sure, I should've looked at it before. Thanks for the link! I've updated the PR.

Reply:
My apologies, the correct name for the function is post_build_tests(). :(

Reply:
np, updated.